data/hai/hai reveal 7 epilogue.txt

```diff
@@ -0,0 +1,820 @@
+		add fleet "Small Syndicate" 1500
+		add fleet "Large Syndicate" 3000
+
+
+mission "HRF 09 Escort Hai Outfitters"
+	landing
+	name "Mooncreek Construction"
+	description "Escort outfitters to build up Mooncreek station."
+	source "Mountaintop"
+	destination "Mooncreek"
+	passengers 1
+	on visit
+		dialog `You have arrived without the convoy you are escorting. You must take off and wait for it to continue.`
+	to offer
+		has "HRF 08 Escort Hai Outfitters: done"
+	on offer
+		conversation
+			`The Hai on Mountaintop are fully prepared. Everyone is eager to get this part over and done with.`
+				accept
+	npc accompany save
+		personality escort timid
+		government "Hai (Wormhole Access)"
+		ship Geocoris "Du Cor Hey"
+		ship Geocoris "Mon Yo Sen"
+		
+		
+#The Free Worlds demand a tour so they can be seen to be contributing to the wider benefit of humanity. (They're PR hogs.)
+#This will need branches in the event of future alternative campaigns.
+mission "HRF 10 Escort Free World Outfitters"
+	landing
+	name "Mooncreek Construction"
+	description "Escort outfitters to build up Mooncreek station."
+	source "Mooncreek"
+	destination "Solace"
+	passengers 1
+	to offer
+		has "HRF 09 Escort Hai Outfitters: done"
+	on offer
+		event "mooncreek main outfitters" 15
+		conversation
+			`On your arrival to Mooncreek with the Hai convoy, you are surprised to see a slightly put-out looking Alondo waiting for you at the docks.`
+			`	"<first>! How's it going?"`
+			choice
+				`	"Fine, I guess?"`
+				`	"It's going well."`
+			`	"Good, good," he says, "now I know that the last thing you probably want to do right now is traipse from one end of space to the other, but I need to ask you a favour."`
+			`	He finds somewhere to sit down with you and begins, "So you may recall I was helping the Free Worlds set up their diplomatic corps. Obviously I wanted no part of the ambassadorial business to the Republic, but when all this Hai business started up well, let's just say they struggled to find anyone qualified. As it turns out someone in the senate doesn't particularly like me and they've been using my extended time away up here to try and influence the diplomatic corps. Meanwhile, the Hai want the Free Worlds to have some involvement with this station we're on so that everyone is tied in, but obviously the Free Worlds don't particularly care when they could hardly be further from it."`
+			`	"I have devised a compromise, where the Free Worlds will not supply ships to the garrison here but they will instead sponsor both Kraz and Delta V to set up outfitters here, with full upfront costs paid by the Free World government. A defining success will allow me to return to the corps with the influence I need to counter this senator's influence. Otherwise I fear I shall be stuck here on the other side of space for the rest of my career."`
+			`	You look at your friend and you can tell he's waiting for you to accept his premise first, holding something back...`
```

Comment:
```suggestion
			`	"Good, good," he says. "Now, I know that the last thing you probably want to do right now is traipse from one end of space to the other, but I need to ask you a favor."`
			`	He finds somewhere to sit down with you and begins. You may recall that I was helping the Free Worlds set up their diplomatic corps. Obviously, I wanted no part of the ambassadorial business to the Republic, but when all this Hai business started up well, let's just say they struggled to find anyone qualified. As it turns out, someone in the senate doesn't particularly like me, and they've been using my extended time away up here to try and influence the diplomatic corps. Meanwhile, the Hai want the Free Worlds to have some involvement with this station we're on so that everyone is tied in, but obviously the Free Worlds don't particularly care when they could hardly be further from it.`
			`	"I have devised a compromise: the Free Worlds will not supply ships to the garrison, but they will instead sponsor both Kraz and Delta V, primarily based in the south, to set up outfitters here, with full upfront costs paid by the Free Worlds government. A defining success will allow me to return to the corps with the influence I need to counter this senator's influence. Otherwise, I fear I shall be stuck here, on the other side of space, for the rest of my career."`
			`	You look at your friend, and you can tell he's waiting for you to accept his premise first, holding something back...`
```